Lib/enum.py

```diff
@@ -235,6 +236,13 @@ def __new__(metacls, cls, bases, classdict):
             # a DynamicClassAttribute
             if member_name not in base_attributes:
                 setattr(enum_class, member_name, enum_member)
+            # issue34282: Members named _convert should take precedence over
+            # the _convert method except if the enum also has a behavior named
+            # _convert. In which case, it falls back to the default
+            # behavior > member ordering where the behavior takes precedence.
```

Comment:
There should never be a member and a method with the same name.  Besides _convert (which is a bug) are there any other examples?

Reply:
This is what I was referring to:

```py
>>> import enum
>>>
>>> class Base(enum.Enum):
...     def test(self): ...
...
>>> class Test(Base):
...     test = enum.auto()
...
>>> Test.test
<function Base.test at 0x000002090A5AD730>
```

It doesn't seem constricted to just _convert.

Reply:
It's the same bug, though.  Fixing the shadowing bug will resolve this case as well.  `Enum`s are special in many ways, but subclass declarations should still override parent's.

Reply:
I feel that's out of scope for this PR. I think it would be better served to close this and create a separate PR (and issue?) for it against the master branch and backport that to 3.7 and 3.6. We should still keep the _convert rename for 3.8 though.

Reply:
Another approach would be to add it to #8568 and rewrite the code here but I'm not sure if fixing multiple things in a single PR is good practice. The titles and news entries would also have to change.

Reply:
As I said in https://bugs.python.org/msg324415, we'll need a patch for 3.7- to fix the member not showing up, a patch in 3.8 to do a rename and deprecation warning, and a reminder to remove the deprecation and legacy _convert in 3.9.

So use one of these two PRs for one, and the other for the other.  :)

Reply:
The latter two are fine but if we're going to fix the member not showing up by fixing the shadowing bug shouldn't it also be added to #8568? It affects more than just _convert so having the above code snippet change for 3.7-3.6 and then revert in 3.8 would be strange.

Reply:
Yes, the actual fix from the 3.7 patch needs to also be in the 3.8 patch.